presto-main/src/main/java/io/prestosql/sql/planner/plan/JoinNode.java

```diff
@@ -52,6 +52,7 @@
     private final List<EquiJoinClause> criteria;
     private final List<Symbol> leftOutputSymbols;
     private final List<Symbol> rightOutputSymbols;
+    private final boolean cardinalityInsensitive;
```

Comment:
Would that allow us to remove special treatment of streaming aggregations now?

Reply:
I don't think so. Streaming aggregations use constant memory and can aggregate actual functions.

Reply:
But I think we can improve correlated `EXISTS(... corr_x < y)` subqueries